x-pack/plugins/ml/public/application/routing/routes/aiops/log_categorization.tsx

```diff
@@ -46,19 +38,14 @@ export const logCategorizationRouteFactory = (
   disabled: !AIOPS_ENABLED,
 });
 
-const PageWrapper: FC<PageProps> = ({ location, deps }) => {
-  const { redirectToMlAccessDeniedPage } = deps;
-
-  const { index, savedSearchId }: Record<string, any> = parse(location.search, { sort: false });
-  const { context } = useResolver(index, savedSearchId, deps.config, deps.dataViewsContract, {
-    checkBasicLicense,
-    cacheDataViewsContract: () => cacheDataViewsContract(deps.dataViewsContract),
-    checkGetJobsCapabilities: () => checkGetJobsCapabilitiesResolver(redirectToMlAccessDeniedPage),
-  });
+const PageWrapper: FC = () => {
+  const { context } = useRouteResolver('basic', []);
```

Comment:
the license check here should be for `full` as this feature uses a platinum licensed aggregation.
I am unsure about the other two AIOPs features.

The[ nav menu links](https://github.com/elastic/kibana/blob/70472fd42a6a673b9eba48c363e3425ad1a86d58/x-pack/plugins/ml/public/application/components/ml_page/side_nav.tsx#LL255C25-L255C25) to these features checks for a full license, so at the moment these pages are very hard get to when running in basic.